### DISCUSSION TITLE
Timeout issue

### DISCUSSION THREAD

**Discussion by Happ1ness-dev** [jing332/tts-server-android#218] [Q&A]
Hello, I'm having issues with the TTS server. Specifically the `ttsrv.httpPost` method. It's timing out after about 10 seconds if it doesn't receive the response in that timeframe. I've tried everything in my power to get around that, even using a proxy which would send keep-alive headers and packets, but it doesn't help.

I'm out of ideas. Changing timeout time in the settings of the TTS server doesn't actually do anything for me.

P.S.
I don't know why exactly, but the issue tab for this project is just gone now, so this is the only place where I can ask for help.

**Comment by amaiaya** [comment]
Same issue here.
I tried the 1.24 dev version and found that the custom TTS using HTTP no longer exists for no reason.
